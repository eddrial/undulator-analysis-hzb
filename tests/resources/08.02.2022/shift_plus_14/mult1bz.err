 error report for the data set mult1bz.dvm         
--------------------------------------------
      
         128   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1590 data points 
        out of totally   2557 data points
 the rms value of the residuals is  0.23201E-02
 the maximum residual is  0.90415E-02
 the rms value of the lambda/2-filtered data is  0.16778E-01
 the maximum value of the lambda/2-filtered data is  0.11648E+00
