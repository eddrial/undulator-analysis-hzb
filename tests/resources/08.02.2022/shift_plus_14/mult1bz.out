mult1bz.dvm 
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -5.0973141E-01
   2  6.9681829E-04
   3  2.9271608E-03
   4 -2.9517856E-05
   5 -7.2619390E-05
   6  5.5981873E+01
   7  1.0389591E+03
   8  1.0574216E+03
   9  1.0445511E+03
  10 -2.9562075E+01
  11  1.3174000E-01
 number of iterations:    9
     sumsq:  8.55859E-03
       rms:  2.32008E-03  dimension of magnetic field
       rms:  4.55157E-01  percent of b_eff
   rms_ext:  4.62947E-01  percent of b_eff
 B_eff =   5.0973E-01
 B_max =  -5.1201E-01
 K_eff =    2.666
 lamda_1 (nm) =     11.56857

        eigenvectors in parameter space

 error                         direction
     3.56,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     1.13,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     0.35,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.05,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00, -0.21  0.49  0.63 -0.56  0.07  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.63 -0.39 -0.12 -0.17  0.64  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.46  0.41 -0.50 -0.32  0.52  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.26  0.56  0.08  0.72  0.30  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.53 -0.35  0.57  0.20  0.48  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:   0.11720 Tmm
 kick_2:  -0.11720 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.18437E+02
      
 the truncated data set has only   1590 data points 
      
 the rms value of the residuals is  0.23201E-02
 the maximum residual is  0.90415E-02
      
 the rms value of the lambda/2-filtered data is  0.16778E-01
 the maximum value of the lambda/2-filtered data is  0.11648E+00
      
 the rms phase error (all data) is    3.354 deg.
 the maximum phase error (all data) is    9.181 deg.
      
 the rms phase error (extrema) is    3.741 deg.
 the maximum phase error (extrema) is    9.282 deg.
      
 the Walker-rms phase error (all data) is   16.164 deg.
 the Walker-maximum phase error (all data) is   30.454 deg.
      
 the Walker-rms phase error (extrema) is    3.712 deg.
 the Walker-maximum phase error (extrema) is    8.700 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             3.353540    
 local phase error             1.882700    
 slowly varying phase error    2.492730    
 oscillatory phase error       1.219823    
 -----------------------------------------
